packages/cli/src/databases/entities/installed-nodes.ts

```diff
@@ -10,7 +10,7 @@ export class InstalledNodes {
 	type: string;
 
 	@Column()
-	latestVersion: string;
+	latestVersion: number;
```

Comment:
This change is kinda unrelated to this PR. Could it be extracted into its own?

Reply:
https://github.com/n8n-io/n8n/pull/10782